docs/portal-builder/maps/discarded.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Discarded
+---
+
+{{< toc >}}
+
+# Discarded
+
+![Discarded thumbnail](../images/thumbnails/discarded_thumbnail.jpg)
+
+## Discarded - Breakthrough (core)
+
+{{< tabs "Discarded-Breakthrough-core" >}}
+{{< tab "standard" >}}
+### Discarded - Breakthrough (core) - standard
+
+_Map Layout is not supported in portal or not yet documented by the community!_
+{{< /tab >}}
+{{< /tabs >}}
+## Discarded - Breakthrough Large (core)
+
+{{< tabs "Discarded-Breakthrough Large-core" >}}
+{{< tab "standard" >}}
+### Discarded - Breakthrough Large (core) - standard
+
+_Map Layout is not supported in portal or not yet documented by the community!_
+{{< /tab >}}
+{{< /tabs >}}
+## Discarded - Conquest (core)
+
+{{< tabs "Discarded-Conquest-core" >}}
+{{< tab "standard" >}}
+### Discarded - Conquest (core) - standard
+
+![Map Layout of Discarded - core - Conquest - standard](../images/layouts/discarded_conquest_core_standard.png){{< /tab >}}
```

Comment:
Is there any specific reason to keep the image images in a separate folder, rather than making a specific folder for each map? 
like how we do it in https://github.com/battlefield-portal-community/portal-docs/tree/main/docs/tutorial/Set-Score-and-Time-Target

that way the image and assets of a map are contained in its own directory which should lead to fewer chances of accidental overwrites